glib-compile-schemas and gtk-update-icon-cache should not be run in src_
phases, they are run in pkg_post functions

So patch them out from meson.build

Bug: https://bugs.gentoo.org/890062

--- a/meson.build
+++ b/meson.build
@@ -89,11 +89,6 @@
 gmodule_dep = dependency('gmodule-2.0')
 gcr_dep = dependency('gcr-3', version: '>= 3.5.5')
 
-meson.add_install_script(
-  join_paths('build-aux', 'post_install.py'),
-  datadir
-)
-
 # Both heimdal and mit ship a krb5.pc:
 krb5_dep    = dependency('krb5')
 
